devices.js

```diff
@@ -1154,6 +1154,50 @@ const devices = [
             await bind(device.getEndpoint(2), coordinatorEndpoint, ['genOnOff']);
         },
     },
+    {
+        zigbeeModel: ['TS0003'],
+        model: 'TS0003',
+        vendor: 'TuYa',
+        description: '3 gang switch',
+        whiteLabel: [
+            {vendor: 'Tuya', model: 'TS0003'},
+            {vendor: 'Zemismart', model: 'ZM-L03E-Z'},
+        ],
+        supports: 'on/off',
+        fromZigbee: [fz.on_off],
+        toZigbee: [tz.on_off],
+        endpoint: (device) => {
+            return {'l1': 1, 'l2': 2, 'l3': 3};
```

Comment:
To prevent a breaking change, you can also add `'left': 1, 'center': 2, 'right': 3` here (it doesn't matter if endpoints are specified multiple times).